src/quoteoftheday/routes.py

```diff
@@ -41,6 +39,15 @@ def index():
 
```

Comment:
Can you please point me where the context is consumed?

Reply:
Anywhere you see ``{{something}}`` in the UI. For example, this is the greeting we show. https://github.com/Azure-Samples/quote-of-the-day-python/blob/7c69f1786eb00316773edbbdc3e950900ae5bbdc/src/templates/index.html#L7